nixos/modules/services/networking/unifi.nix

```diff
@@ -128,16 +128,15 @@ in
         Type = "simple";
         ExecStart = "${cmd} start";
         ExecStop = "${cmd} stop";
+        # Send the signal to the main process so it can clean up its child processes.
+        KillMode = "mixed";
```

Comment:
unifi8 test:
```
server # [   13.307317] systemd[1]: Stopping UniFi controller daemon...
server # [   16.384249] systemd[1]: unifi.service: Killing process 869 (mongod) with signal SIGKILL.
server # [   16.384867] systemd[1]: unifi.service: Failed to kill control group /system.slice/unifi.service, ignoring: Invalid argument
server # [   16.386917] systemd[1]: unifi.service: Deactivated successfully.
server # [   16.387242] systemd[1]: Stopped UniFi controller daemon.
```

Apparently mongod is still running after the main process terminated. We shouldn't send a SIGKILL to a database - at least not after less than 1s. Why is `KillMode` needed? I think the old behavior with the noop SIGCONT `KillSignal` is preferable.

Reply:
I also encountered this kind of problem when debugging ollama. Maybe we should try more signals, such as SIGTERM

Reply:
Thanks for the suggestions, everyone. I'll see if I can come up with something that solves this. 😁

I've also been looking at the official Debian packages, but it seems like they start the service a little differently.